### PR TITLE
ci: identify codesandbox workflow

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: platforms/codesandbox/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "prestart": "CI=1 yarn prisma migrate up --auto-approve --experimental",
+    "preinstall": "export CI=1 && export PRISMA_TELEMETRY_INFORMATION='e2e-tests platform codesandbox preinstall'",
+    "prestart": "CI=1 && PRISMA_TELEMETRY_INFORMATION=\"e2e-tests platform codesandbox prestart\" && yarn prisma migrate up --auto-approve --experimental",
```
Comment (janpio) [inline platforms/codesandbox/package.json]: I don't think this is actually working.

File: platforms/codesandbox/prisma/.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CI=1
```
Comment (janpio) [inline platforms/codesandbox/prisma/.env]: This somehow also does not work :/

Comment (divyenduz) [reply platforms/codesandbox/prisma/.env]: Same, the container did get the correct `.env` file https://codesandbox.io/s/j6txg?file=/prisma/.env

File: platforms/codesandbox/package.json (hunk 1 above)
Comment (divyenduz) [reply platforms/codesandbox/package.json]: The container has the same package.json and that is where `npm install` is run, so it should work https://codesandbox.io/s/j6txg?file=/package.json

File: platforms/codesandbox/prisma/.env (hunk 2 above)
Comment (janpio) [reply platforms/codesandbox/prisma/.env]: No events with `CI=true` for `%codesandbox%`:
https://app.redash.io/prismaio/queries/469381?p_ci=%5B%22false%22%2C%22true%22%5D&p_information=codesandbox

File: platforms/codesandbox/package.json (hunk 1 above)
Comment (janpio) [reply platforms/codesandbox/package.json]: Unfortunately only the value from `.env` is being picked up: 
https://app.redash.io/prismaio/queries/469381?p_ci=%5B%22false%22%2C%22true%22%5D&p_information=codesandbox

Comment (janpio) [reply platforms/codesandbox/package.json]: Removed from the file to not create wrong expectations.

Comment (janpio) [reply platforms/codesandbox/package.json]: Removed from the file to not create wrong expectations.

File: platforms/codesandbox/prisma/.env (hunk 2 above)
Comment (janpio) [reply platforms/codesandbox/prisma/.env]: Added comment to indicate this does not work.

Comment (janpio) [reply platforms/codesandbox/prisma/.env]: https://github.com/prisma/e2e-tests/issues/827

**Pull request merged**